mediaire_toolbox/transaction_db/transaction_db.py

```diff
@@ -236,12 +236,12 @@ def create_transaction(self,
     @lock
     def get_transaction(self, id_: int) -> Transaction:
         try:
-            return self._get_transaction_or_raise_exception(id_)
-        finally:
-            # we should always complete the lifetime of the connection,
-            # otherwise we might run into timeout errors
-            # (see https://docs.sqlalchemy.org/en/latest/orm/session_transaction.html)  # noqa: 501
+            t = self._get_transaction_or_raise_exception(id_)
             self.session.commit()
+            return t
```

Comment:
I think here the lifecycle was not handled properly so I got errors (but in sqlite it doesn't matter because there are no connections)